view-component/src/main/java/com/islamversity/view_component/ext/View.kt

```diff
@@ -0,0 +1,13 @@
+package com.islamversity.view_component.ext
+
+import android.view.View
+
+fun View.hide(){
+    visibility = View.INVISIBLE
+}
+fun View.show(){
+    visibility = View.VISIBLE
+}
+fun View.gone(){
+    visibility = View.GONE
+}
```

Comment:
I would suggest to let's move this to `base` module
so all of our features can see them